tests/storage/restore.go

```diff
@@ -546,7 +546,7 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 					Expect(newVMInterfaces[0].MacAddress).ToNot(Equal(oldVMInterfaces[0].MacAddress))
 
 					By("Making sure new VM is runnable")
-					libvmops.RunVMAndExpectLaunchWithRunStrategy(virtClient, newVM, v1.RunStrategyAlways)
+					Expect(libvmops.StartVirtualMachine(newVM)).ToNot(BeNil())
```

Comment:
I do not understand what this assert is checking.
The result will always be non-nil if the start call does not blow up on an inner assert.

Reply:
Just making sure to avoid leaving an unhandled return, is not very helpful but I think it's good practice.